jvm-observ-lib/signals/hikari.libsonnet

```diff
@@ -9,7 +9,8 @@ function(this)
     aggFunction: 'avg',
     discoveryMetric: {
       java_micrometer: 'hikaricp_connections',  // https://github.com/brettwooldridge/HikariCP/blob/dev/src/main/java/com/zaxxer/hikari/metrics/micrometer/MicrometerMetricsTracker.java
-      prometheus: '?',
+      // prometheus: '?',
+      // prometheus_old: '?',
```

Comment:
Just a question here, if we're not using these discovery metrics, why have they been added to and then commented out?

Reply:
wanted to highlight that haven't found them. Removing for cleanliness.